packages/blockchain-wallet-v4-frontend/src/scenes/Goals/index.js

```diff
@@ -22,8 +22,8 @@ class ActionsContainer extends React.PureComponent {
         const decodedPayload = decodeURIComponent(payload)
         const bip21Payload = bip21.decode(decodedPayload)
         const { address } = bip21Payload
-        const { amount, label } = bip21Payload.options || {}
-        const data = { address, amount, message: label }
+        const { amount, message } = bip21Payload.options || {}
+        const data = { address, amount, description: message }
```

Comment:
Well, I believe these can also have labels

Reply:
what should it do with them?

Reply:
we can't set description to both of them